CLOUD/Get-AWSSizingInfo.ps1

```diff
@@ -780,6 +918,55 @@ function getAWSData($cred) {
       }
       $backupPlanList.Add($BackupPlanObject) | Out-Null
     }
+  }
+  
+  $filter = @{
+    Dimensions = @{
+        Key = "SERVICE"
+        Values = @("AWS Backup")
+    }
+  }
+
+  # Create a date interval for past 12 months
+  $startDate = (Get-Date).AddMonths(-12).ToString("yyyy-MM-01")
+  $endDate = (Get-Date).ToString("yyyy-MM-dd")
+  $timePeriod = @{
+      Start = $startDate
+      End = $endDate
+  }
+
+  $metrics = @("AmortizedCost", "BlendedCost", "NetAmortizedCost", "NetUnblendedCost", "NormalizedUsageAmount", "UnblendedCost", "UsageQuantity")
+
+  Write-Host "Getting Cost and Usage of Backup in Account: $awsAccountAlias" -ForegroundColor Green
+  Write-Progress -Activity 'Processing Cost and Usage of Backup:' -Status $plan.BackupPlanId -PercentComplete (0)
```

Comment:
This line causes an error when there are 0 backup plans. The error is:

`Write-Error: Cannot validate argument on parameter 'Status'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.`

Reply:
@DamaniN resolved both of these errors